erpnext/accounts/general_ledger.py

```diff
@@ -163,9 +163,16 @@ def validate_account_for_perpetual_inventory(gl_map):
 						.format(account), StockAccountInvalidTransaction)
 
 			elif account_bal != stock_bal:
-				frappe.throw(_("Account Balance ({0}) and Stock Value ({1}) is out of sync for account {2} and linked warehouse ({3}). Please create adjustment Journal Entry for amount {4}.")
-					.format(account_bal, stock_bal, account, comma_and(warehouse_list), stock_bal - account_bal),
-					StockValueAndAccountBalanceOutOfSync)
+				frappe.throw(
+					_("""<b>Account Balance</b> and <b>Stock Value</b> is out of sync for account <b>{0}</b> and it's linked warehouses.
```

Comment:
- You have to split this message to translate this properly, otherwise that html code will also be translated.
- Show account and stock value as well

Reply:
done